packages/translate/lib/__tests__/testLineResultThenWarn.test.tsx

```diff
@@ -0,0 +1,971 @@
+import {
```

Comment:
It seems that those files for different "then"s are copy-paste? Can't we have one file and just execute it 3 times with different argument?